tools/setup/emoji/build_emoji

```diff
@@ -153,6 +155,9 @@ def code_point_to_file_name_map(ttx):
 
 def main():
     # type: () -> None
+    run(["sudo", "mkdir", "-p", EMOJI_CACHE_PATH])
+    run(["sudo", "chown", "%s:%s" % (user_id, user_id), EMOJI_CACHE_PATH])
```

Comment:
I think it would be worth adding a comment next to the lines in `provision` related to this; this isn't the first time someone has tried to make that incorrect simplification.